framework/contracts/account/tests/snapshots/subaccount__creating_on_subaccount_should_succeed.snap

```diff
@@ -63,49 +57,43 @@ expression: all_storage
   - - ownership
-    - "{\"owner\":\"mock1pgm8hyk0pvphmlvfjc8wsvk4daluz5tgrw6pu5mfpemk74uxnx9qwrtv4f\",\"pending_owner\":null,\"pending_expiry\":null}"
+    - "{\"owner\":\"mock16g2rahf5846rxzp3fwlswy08fz8ccuwkgsny9p\",\"pending_owner\":null,\"pending_expiry\":null}"
 "abstract:ibc-host":
-  - - cfg
-    - "{\"ans_host\":{\"address\":\"mock1mzdhwvvh22wrt07w59wxyd58822qavwkx5lcej7aqfkpqqlhaqfsetqc4t\"},\"module_factory_addr\":\"mock1qg5ega6dykkxc307y25pecuufrjkxkaggkkxh7nad0vhyhtuhw3sep3g9s\",\"version_control\":{\"address\":\"mock1wug8sewp6cedgkmrmvhl3lf3tulagm9hnvy8p0rppz9yjw0g4wtqwm38hv\"}}"
   - - contract_info
     - "{\"contract\":\"abstract:ibc-host\",\"version\":\"0.23.0\"}"
   - - ownership
-    - "{\"owner\":\"mock1pgm8hyk0pvphmlvfjc8wsvk4daluz5tgrw6pu5mfpemk74uxnx9qwrtv4f\",\"pending_owner\":null,\"pending_expiry\":null}"
+    - "{\"owner\":\"mock16g2rahf5846rxzp3fwlswy08fz8ccuwkgsny9p\",\"pending_owner\":null,\"pending_expiry\":null}"
 "abstract:module-factory":
-  - - cfg
-    - "{\"version_control_address\":\"mock1wug8sewp6cedgkmrmvhl3lf3tulagm9hnvy8p0rppz9yjw0g4wtqwm38hv\",\"ans_host_address\":\"mock1mzdhwvvh22wrt07w59wxyd58822qavwkx5lcej7aqfkpqqlhaqfsetqc4t\"}"
   - - contract_info
     - "{\"contract\":\"abstract:module-factory\",\"version\":\"0.23.0\"}"
   - - ownership
-    - "{\"owner\":\"mock1pgm8hyk0pvphmlvfjc8wsvk4daluz5tgrw6pu5mfpemk74uxnx9qwrtv4f\",\"pending_owner\":null,\"pending_expiry\":null}"
+    - "{\"owner\":\"mock16g2rahf5846rxzp3fwlswy08fz8ccuwkgsny9p\",\"pending_owner\":null,\"pending_expiry\":null}"
 "abstract:version-control":
   - - "\u0000\u0002cb\u0000\babstract\u0000\u0007account0.23.0"
-    - "{\"account\":4}"
+    - "{\"account\":5}"
   - - "\u0000\u0002cb\u0000\babstract\u0000\bans-host0.23.0"
-    - "{\"native\":\"mock1mzdhwvvh22wrt07w59wxyd58822qavwkx5lcej7aqfkpqqlhaqfsetqc4t\"}"
+    - "{\"native\":\"mock17r8ljc3tw8tdx85gpwxpq5ten89622svh2ct0utcg75x89ed7cgsyz0g62\"}"
   - - "\u0000\u0002cb\u0000\babstract\u0000\bibc-host0.23.0"
-    - "{\"native\":\"mock1466nf3zuxpya8q9emxukd7vftaf6h4psr0a07srl5zw74zh84yjqqk0lq9\"}"
+    - "{\"native\":\"mock19jf2y29m2qvgfezcyy5zhq8phfdcysngew508eeumdsg2dyzqrqshlx44z\"}"
   - - "\u0000\u0002cb\u0000\babstract\u0000\nibc-client0.23.0"
-    - "{\"native\":\"mock1ufs3tlq4umljk0qfe8k5ya0x6hpavn897u2cnf9k0en9jr7qarqqgfzm3q\"}"
+    - "{\"native\":\"mock1u377ag8ewzrskj6r0pqutnpyvh7hd2yqq6uh3rh2pclex2gzhslsq3frn0\"}"
   - - "\u0000\u0002cb\u0000\babstract\u0000\u000emodule-factory0.23.0"
-    - "{\"native\":\"mock1qg5ega6dykkxc307y25pecuufrjkxkaggkkxh7nad0vhyhtuhw3sep3g9s\"}"
+    - "{\"native\":\"mock10ymkph7y47lmxkyan49cq2yjm5m3n2nk83efppcwwpunx7gs374qg8fkqs\"}"
   - - "\u0000\u0002cb\u0000\babstract\u0000\u000fversion-control0.23.0"
-    - "{\"native\":\"mock1wug8sewp6cedgkmrmvhl3lf3tulagm9hnvy8p0rppz9yjw0g4wtqwm38hv\"}"
+    - "{\"native\":\"mock18aq2nqrm809gdktwetuk6c48c6yj379ptkdeewv6j797xv3j2gssdzgurv\"}"
```

Comment:
@CyberHoward We already save those addresses inside all snapshot tests. Address change is also covered by `deploy2` test